lte/gateway/c/core/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -271,12 +278,21 @@ status_code_e send_pcrf_bearer_actv_rsp(
 
   message_p->ittiMsgHeader.imsi = ue_context_p->emm_context._imsi64;
 
-  OAILOG_INFO_UE(
-      LOG_MME_APP, ue_context_p->emm_context._imsi64,
-      "Sending create_dedicated_bearer_rsp to SGW with EBI %u s1u teid %u for "
-      "ue id " MME_UE_S1AP_ID_FMT "\n",
-      ebi, bc->s_gw_fteid_s1u.teid, ue_context_p->mme_ue_s1ap_id);
-  send_msg_to_task(&mme_app_task_zmq_ctx, TASK_SPGW, message_p);
+  if (pdn_context->route_s11_messages_to_s8_task) {
+    OAILOG_INFO_UE(
+        LOG_MME_APP, ue_context_p->emm_context._imsi64,
+        "Sending create_dedicated_bearer_rsp to SGW_s8 task for EBI %u s1u "
```

Comment:
is `SGW_s8` the standard name we use to identify that task in other log files?

Reply:
Should we add the IMSI besides the EBI on the log so we can identify what IMSI is being created. Or is the IMSI printed somewhere else during this process? or is `MME_UE_S1AP_ID_FMT` the IMSI itself ?

Reply:
Imsi is logged as part of OAILOG_INFO_UE.

Reply:
> is `SGW_s8` the standard name we use to identify that task in other log files?

Create bearer response is sent from MME; So included LOG_MME_APP is included.
If any handling is performed in context of SGW_S8 task, while logging LOG_SGW_S8 is used.